### PR TITLE
Including PT-BR translation to the bot.

### PR DESCRIPTION
**Pull request by elaynelemos**
This is the link to the test bot: [t.me/evm00dbot](https://t.me/evm00dbot)

During the development I had trouble installing the seaborn module but I solved it through the following package:
`pip3 install git+https://github.com/mwaskom/seaborn.git` 
I would include this tip at the read me file but I don't know if you would like. Then, it's up to you.

### CODE REVIEW

File: quizes.py [hunk 1]
```diff
@@ -74,7 +75,8 @@ class MADRSQuiz(BaseQuiz):
 
     RESULTS = {
         'en': ['normal 👍', 'mild depression 😐', 'moderate depression 😔', 'severe depression 😨'],
-        'ru': ['норма 👍', 'слабая депрессия 😐', 'умеренная депрессия 😔', 'тяжелая депрессия 😨']}
+        'ru': ['норма 👍', 'слабая депрессия 😐', 'умеренная депрессия 😔', 'тяжелая депрессия 😨'],
+        'en': ['normal 👍', 'depressão leve 😐', 'depressão moderada 😔', 'depressão severa 😨']}
```
Comment (dizballanze) [inline quizes.py]: Wrong key, there should be `pt` instead of `en`

**Comment by dizballanze** [comment]
Hi! Thank you for your PR. There is a small typo in code. Fix it please and I'll deploy new version.

File: quizes.py [hunk 2]
```diff
@@ -41,7 +41,8 @@ class HARSQuiz(BaseQuiz):
 
     RESULTS = {
         'ru': ['отсутствие тревоги 👍', 'средняя выраженность тревожного расстройства 😐', 'тяжелая тревога 😦'],
-        'en': ['mild anxiety severity', 'mild to moderate anxiety severity', 'moderate to severe anxiety severity']}
+        'en': ['mild anxiety severity 👍', 'mild to moderate anxiety severity 😐', 'moderate to severe anxiety severity 😦'],
+        'pt': ['ansiedade leve 👍', 'ansiedade leve a moderada 😐', 'ansiedade moderada a severa 😦']}
```
Comment (elaynelemos) [inline quizes.py]: Fixed language key previous conflict.

File: quizes.py (hunk 1 above)
Comment (elaynelemos) [reply quizes.py]: Fixed language key previous conflict.

**Comment by dizballanze** [comment]
Hi. I've deployed your changes on production. Thank you, again!

**Pull request merged**